### PR TITLE
openssl_certificate: consistent param namings

### PR DESCRIPTION
**Pull request by resmo**
##### SUMMARY
the module's params were mixed case CamelCase_snake_case. I changed it to more consistent namings by adding aliases.

##### ISSUE TYPE

 - Bugfix Pull Request

##### COMPONENT NAME
openssl_certificate

##### ANSIBLE VERSION

```
2.4
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

File: lib/ansible/modules/crypto/openssl_certificate.py [hunk 1]
```diff
@@ -243,32 +253,32 @@
...
 - name: Ensure that the existing certificate can only be used for client authentication and time stamping
   openssl_certificate:
     path: /etc/ssl/crt/example.com.crt
     provider: assertonly
-    extendedKeyUsage:
+    extended_key_usage:
       - clientAuth
       - 1.3.6.1.5.5.7.3.8
-    extendedKeyUsage: strict
+    extended_key_usage_strict: true
```
Comment (resmo) [inline lib/ansible/modules/crypto/openssl_certificate.py:275]: IMHO this was wrong before, wasn't it?

**Comment by resmo** [comment]
@Spredzy @MarkusTeufelberger any comment?

File: lib/ansible/modules/crypto/openssl_certificate.py (hunk 1 above)
Comment (MarkusTeufelberger) [reply lib/ansible/modules/crypto/openssl_certificate.py:275]: Seems like a leftover from my version, so yes, it likely was wrong.

**Comment by MarkusTeufelberger** [comment]
I like it, no preferences either way, so it might as well be more standardized. :-)

**Comment by MarkusTeufelberger** [comment]
shipit

**Comment by Spredzy** [comment]
Talked about it on IRC, lgtm + its transparent 👍

**Comment by Spredzy** [comment]
shipit

**Comment by MarkusTeufelberger** [comment]
Hm, seems like we overlooked the documentation for notBefore.

**Comment by resmo** [comment]
@MarkusTeufelberger thanks for reporting, fixing

**Pull request merged**